### DISCUSSION TITLE
Moving between file storage locations

### DISCUSSION THREAD

**Discussion by unleashit** [directus/directus#16450] [Archive]
Just wondering if there is an easy way to switch a file's storage location (both physical and DB) after being uploaded? In the GUI, the storage field is locked for me. 

If not, what is the recommended procedure? Is it enough to update the DB's storage field then move the asset to the new location?

Basically, I started developing with local storage and would like to move files to the cloud in production. I've learned that files default to the first storage in the array (or can be selected when uploading via the API), but I'm not sure if it's possible to transfer between after the fact.

**Comment by rijkvanzanten** [comment]
There's no moving functionality built in to Directus, no. You should be able to copy paste all the files to your cloud host, and update the `storage` field directly in the database, but your mileage may vary

**Comment by unleashit** [comment]
Ok, thanks for clarifying. Good to know there isn't too much happening aside from those things, but I think I'll just use the final storage for dev (no biggie).
